Script/Player.gd

```diff
@@ -1,106 +1,105 @@
-class_name Player
-extends CharacterBody2D
+extends CharacterBody2D  # CharacterBody2D gives us built-in physics properties for 2D movement
 
-signal stomped(goober: CharacterBody2D)
-signal died
+signal stomped(goober: CharacterBody2D)  # Sent when player jumps on an enemy
+signal died                              # Sent when player is hit by an enemy
 
 ## If false, the player cannot be moved and does not interact with its
 ## environment: it just performs its idle animation, as needed for a title
 ## or world-complete level.
-@export var interactive := true
-
-@onready var NodeSprite := $Sprite2D
-@onready var NodeArea2D := $Area2D
-@onready var NodeAudio := $Audio
-@onready var NodeAnim := $AnimationPlayer
-
-var vel := Vector2.ZERO
-var spd := 60.0
-var grv := 255.0
-var jumpSpd := 133.0
-var termVel := 400.0
-
-var onFloor := false
-var jump := false
-
-func _ready() -> void:
-	var world_textures := ResourceFinder.load_world_assets(self, "Player")
+@export var interactive := true  # Makes this variable appear in the Inspector panel
```

Comment:
I think this comment is actively misleading. The comment refers to the `@export` annotation but I would read it as saying that the `interactive` property itself has something to do with the inspector panel.